src/H5C.c

```diff
@@ -1310,7 +1310,7 @@ H5C_insert_entry(H5F_t *f, const H5C_class_t *type, haddr_t addr, void *thing, u
     hbool_t     flush_last;
 #ifdef H5_HAVE_PARALLEL
     hbool_t coll_access = FALSE; /* whether access to the cache entry is done collectively */
-#endif                           /* H5_HAVE_PARALLEL */
+#endif /* H5_HAVE_PARALLEL */
```

Comment:
The formatter did this. It will be undone as soon as someone re-runs it.

Reply:
I don't know which switch will give us what we want. However there are some easy (not perfect) solutions.
A blank line resets the comment/equal sign alignments.
Put the previous comment by itself on the line before statement instead of at the end of the line.